snapcraft/internal/sources/__init__.py

```diff
@@ -87,9 +86,11 @@
 
 from snapcraft.internal import common
 from snapcraft import file_utils
-from snapcraft.internal.indicators import (
-    download_requests_stream,
-    download_urllib_source
+
+from ._base import (  # noqa
+    Base,
```

Comment:
It was previously so I continued doing so, but no-- technically not. I've replaced this with an import of _base.